doc/mgr/nfs.rst

```diff
@@ -493,6 +498,8 @@ provided JSON should fully describe the new state of the export (just
 as when creating a new export), with the exception of the
 authentication credentials, which will be carried over from the
 previous state of the export where possible.
+The ``user_id`` in the ``fsal`` block should not be modified or mentioned in the JSON file as it is auto-generated for CephFS exports.
```

Comment:
nit: this should go as a warning.

Reply:
Oh this needs to be a test case, to see what happens if the modifications are made, yes? @avanthakkar @vshankar

Reply:
> Oh this needs to be a test case, to see what happens if the modifications are made, yes? @avanthakkar @vshankar

Not super urgent. If users are shooting themselves in their foot, there is just so much we could do. Given that its documented, we can not bother about this now.